### DISCUSSION TITLE
Font level size control

### DISCUSSION THREAD

**Discussion by samyak-jain** [kovidgoyal/kitty#3391] [Q&A] [answered]
Kitty has ```font_size``` to control the size of all the fonts. Do we have anything to change the size of a particular font? The reason that I'm asking this is because Nerd fonts always look bigger than I'd like when paired up with text. It would be great if I could somehow make Nerd fonts alone slightly smaller.

**Comment by kovidgoyal** [comment, marked as answer]
No there are no such controls.

**Reply by samyak-jain** [reply]
Gotcha. Thanks for the reply. Is this something you are open to merging if someone contributes or is this out of scope? Are you aware of any workarounds I can do on my side?

**Comment by kovidgoyal** [comment]
If I were you I would just choose a main font whose metrics are similar to those of nerd fonts. That said, I dont actually mind having per font size settings, patches are welcome.

**Reply by samyak-jain** [reply]
Is the only way to go about choosing the main font, trial and error? Or is there a more sophisticated way? What's actually happened is that I have install nerd fonts from a better source now and the fonts seem to be properly sized. The only problem is that they don't seem to vertically centered. The nerd fonts seems to be slightly above the actual line of text. Is this also because of difference in the metrics with the main font?

**Reply by kovidgoyal** [reply]
yes trial and error and font baselines are part of ont metrics
